packages/common/src/api/signUp.ts

```diff
@@ -21,15 +21,23 @@ const signUpApi = createApi({
     },
     isHandleInUse: {
       fetch: async ({ handle }: { handle: string }, context) => {
-        const user = await userApiFetch.getUserByHandle(
-          {
-            handle,
-            currentUserId: null,
-            retry: false
-          },
-          context
-        )
-        return !isEmpty(user)
+        try {
+          const user = await userApiFetch.getUserByHandle(
+            {
+              handle,
+              currentUserId: null,
+              retry: false
+            },
+            context
+          )
+          return !isEmpty(user)
+        } catch (e) {
+          if (e instanceof RemoteDataNotFoundError) {
+            window.alert('handle not found')
```

Comment:
alert? intentional or debugging?

Reply:
haha yep you're faster than me. Was just about to remove it

Reply:
in fact, we don't need this try catch at all since the createAPI change should not rethrow